app/main.py

```diff
@@ -16,4 +16,10 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+immutable = [lucky_number, pi, one_is_a_prime_number, name, profile_info]
```

Comment:
The list `immutable` contains elements like `profile_info`, which is a tuple and indeed immutable, but it also contains `name`, which is a string and immutable. However, the list itself is mutable. Consider revising the categorization to ensure that the list accurately represents immutable data structures.

Reply:
Readme doc states that value should be a **list** that contains those variables.